src/main/java/mate/jdbc/model/Manufacturer.java

```diff
@@ -0,0 +1,45 @@
+package mate.jdbc.model;
+
+public class Manufacturer {
```

Comment:
Add a new line here. This is required to separate the class declaration and the first field declaration.